dee-pabst/week4/crud-active-record-site/views/artists/show.erb

```diff
@@ -0,0 +1,10 @@
+<h2><%= @artist.name %></h2>
+<img src="<%= @artist.image_url %>" alt="<%= @artist.name %>">
+<table>
+  <tr>
+    <td>Web</td>
+    <td><a href="<%= @artist.website %>"><%= @artist.website %></a></td>
+  </tr>
+</table>
```

Comment:
you need to make the associative link back to the song. 
You will need to access the association `@artists.hits` and then create an each loop and print out each song. From there you should be able to access the id and title to create a link. if you are still trying to work this out, let me know and i can help!